.azure-pipelines/docker.yml

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the ODPi Egeria project.
+---
+trigger:
+  paths:
+    include:
+      - open-metadata-resources/open-metadata-deployment/docker
+pr: none
+schedules:
+  - cron: "11 0 * * *"
+    displayName: Daily Docker Build
+    always: true
+    branches:
+      include:
+        - master
+
+stages:
+  - stage: Docker
+    jobs:
+      - job: Maven
+        pool:
+          vmImage: 'ubuntu-latest'
+        strategy:
+          matrix:
```

Comment:
In order to improve performance we added some base images under 'build-images'. These are building code from other repos (atlas and ranger) without much customization (that's in the top level images). I'd expect us to only build infrequently as we're using <released> code. I'm just wondering if we can have a manual trigger for those - though an alternative is direct posting to the docker repo (which I've done before)

Reply:
Yeah it wouldn't be much work to pull those out to a separate pipeline that doesn't have the 'scheduled' trigger as well. Then they can be triggered manually from the UI.

Reply:
So for the current 'phase' the proposal is we leave those running in jenkins? Do we need to open a specific issue to track implementing them in Azure pipelines?

Reply:
Yeah it'd be helpful to have the issues written out for how you envision the docker builds working given it seems to be a bit more involved then just running maven.